### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by harimkang**
### Summary

Modify CODEOWNERS at the team's request.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -11,33 +11,33 @@
...
 # OTX Adapters mmengine
-/src/otx/v2/adapters/torch/mmengine                         @samet-akcay @harimkang @goodsong81
+/src/otx/v2/adapters/torch/mmengine                         @samet-akcay @harimkang @goodsong81 @wonjuleee @vinnamkim @jaegukhyun @sungmanc @eugene123tw
 
 # OTX Adapters mmdeploy
 /src/otx/v2/adapters/torch/mmengine/mmdeploy                @samet-akcay @harimkang @goodsong81 @sovrasov
 
 # Tests
-/tests/v2                                                   @samet-akcay @harimkang @goodsong81
+/tests/v2                                                   @samet-akcay @harimkang @goodsong81 @wonjuleee
```
Comment (sungmanc) [inline .github/CODEOWNERS]: How about adding execution member including @jaegukhyun , @eugene123tw , @kprokofi  and me for the tests?

**Review by samet-akcay — APPROVED** [review]
Thanks @harimkang

File: .github/CODEOWNERS (hunk 1 above)
Comment (harimkang) [reply .github/CODEOWNERS]: Thanks, Done!

**Review by sungmanc — APPROVED** [review]
LGTM

**Review by jaegukhyun — APPROVED** [review]

**Pull request merged**